pkg/kapp/cmd/app/apply_flags.go

```diff
@@ -45,6 +45,7 @@ func (s *ApplyFlags) SetWithDefaults(prefix string, defaults ApplyFlags, cmd *co
 		mustParseDuration("15m"), "Maximum amount of time to wait in apply phase")
 	cmd.Flags().DurationVar(&s.ApplyingChangesOpts.CheckInterval, prefix+"apply-check-interval",
 		mustParseDuration("1s"), "Amount of time to sleep between applies")
+	cmd.Flags().BoolVar(&s.ApplyingChangesOpts.Run, prefix+"apply-run", false, "Print new generated YAML to be applied and exit.")
```

Comment:
@vmware-tanzu/carvel-ind What do you all think about calling it `--changes` and then it can used with (or without) `--diff-run`, just like `--diff-chages`

Reply:
nit:`Print new generated YAML to be applied and exit.` . I think we can make it more clear. May be `Print YAML to be applied and exit.`